rwd_nhd/NHD_Rapid_Watershed_Delineation.py

```diff
@@ -25,9 +26,11 @@ def Point_Watershed_Function(
 
     start_time = time.time()
 
-    dir_main = os.path.join(str(pre_process_dir), 'Main_Watershed')
+    dir_main = pre_process_dir
```

Comment:
For the sake of keeping these files grouped together, and to maintain parity with the DRB folder layout, I'd like to consider reverting this change.